sql/core/src/test/resources/sql-tests/inputs/pgSQL/comments.sql

```diff
@@ -0,0 +1,48 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- COMMENTS
+-- https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/comments.sql
+--
+
+SELECT 'trailing' AS first; -- trailing single line
+SELECT /* embedded single line */ 'embedded' AS `second`;
+SELECT /* both embedded and trailing single line */ 'both' AS third; -- trailing single line
+
+SELECT 'before multi-line' AS fourth;
+-- [SPARK-28880] ANSI SQL: Bracketed comments
+/* This is an example of SQL which should not execute:
```

Comment:
Do we need to comment these unsupported cases?

Reply:
I think its ok to comment out this case.